clients/apple/Shared/Stateful Logic/DocumentService.swift

```diff
@@ -282,7 +284,17 @@ class DocumentLoadingInfo: ObservableObject {
             DispatchQueue.main.async {
                 switch operation {
                 case .success(let txt):
-                    self.textDocument = EditorState(text: txt, isiPhone: self.isiPhone)
+                    self.textDocument = EditorState(text: txt, isiPhone: self.isiPhone) { url in
+                        DI.importExport.importFilesSync(sources:[url.path(percentEncoded: false)], destination: self.meta.parent)
+                        
+                        if let parentPath = DI.files.getPathByIdOrParent(maybeId: self.meta.parent) {
+                            if let file = DI.files.getFileByPath(path: parentPath + url.lastPathComponent) {
+                                return "![alt text](lb://\(file.id.uuidString.lowercased()))"
```

Comment:
I wonder if we could have better alt text

Reply:
I changed the alt name to the image's name (minus the extension).